### ISSUE TITLE
`PoolPair` Service/Controller Extra Stats

### ISSUE THREAD

**Issue by fuxingloh** [DeFiCh/whale#136]
#### What would you like to be added:

Extending on top of the current `PoolPairController` via `PoolPairService`, we need to display statistical data similar to how it's done in https://dex.defichain.com/mainnet/pool but in a decentralized implementation that is resilient to network cascade.

- Price Ratio
- APR
- Liquidity (USD)
- Daily Volume (USD)
  - 30 Day Average
  - 1 Day 
- Total Liquidity (Pairs)

We don't need external service for this, (USD) can be calculated by the USDT-DFI pair. We can derive all value from that pair in real-time. As `USDT` pair configuration can vary in different networks and for the purpose of testing, we should be able to set that value in `configuration.ts`. This allows us to use varying base pair for different currency calculation as the base.

This should be implemented with caching, (blocked by #82) and the logic should be separated into PoolPairService for good practices. 

/triage accepted
/assign @canonbrother 
/priority important-soon

Areas of concerns are, but might not be limited to:
/area module-api module-indexer module-database module-model
Note that there is technical debt in the `module-indexer` and `module-model` due to the crunch, they are not the cleanest code.

#### Why is this needed:

For Wallet & Explorer to display statistical data.

## `sub stories`
- [x] add priceRatio and totalLiquidityUsd
- [ ] poolswap block sync, aggregation and index
- [ ] add apy and volume (avg 30 days)

**Comment by canonbrother** [comment]
@monstrobishi 
would you know how does the `APR` and `Daily Volume` calculate? (previously i thought the `APR` is hardcoded but its not)
https://dex.defichain.com/mainnet/pool
![image](https://user-images.githubusercontent.com/8097901/121873182-b09eec80-cd38-11eb-8c85-fa831c4589f0.png)

**Comment by monstrobishi** [comment]
Reference from stats API, APR for custom rewards is calculated here: 
https://github.com/cakedefi/defi-stats-api/blob/930ef203d3a3acd07c9c69f5daf48ca4c1075d5b/src/config/utility/utility.service.ts#L114

APR for daily rewards is calculated here:
https://github.com/cakedefi/defi-stats-api/blob/930ef203d3a3acd07c9c69f5daf48ca4c1075d5b/src/config/utility/utility.service.ts#L128

The two are then combined

For volume, all swap tx are stored in a db, then summed up and averaged out here:
https://github.com/cakedefi/defi-stats-api/blob/930ef203d3a3acd07c9c69f5daf48ca4c1075d5b/src/config/utility/utility.service.ts#L91

**Comment by fuxingloh** [comment]
For stats that doesn't require block aggregation, should create a separate PR for it first. I need that soon.

That would be:
- Price Ratio, e.g. `3.04 USDT/DFI`
- Liquidity (USD), e.g. `13,863,592.73 USD`

**Comment by fuxingloh** [comment]
Completed
